frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-inclusions-section-card/feature-flag-inclusions-section-card.component.ts

```diff
@@ -29,18 +33,68 @@ import { DialogService } from '../../../../../../../shared/services/common-dialo
 export class FeatureFlagInclusionsSectionCardComponent {
   @Input() isSectionCardExpanded;
   tableRowCount$ = this.featureFlagService.selectFeatureFlagInclusionsLength$;
+  selectedFlag$ = this.featureFlagService.selectedFeatureFlag$;
 
-  constructor(private featureFlagService: FeatureFlagsService, private dialogService: DialogService) {}
+  rowCountWithInclude$: Observable<number> = combineLatest([this.tableRowCount$, this.selectedFlag$]).pipe(
+    map(([tableRowCount, selectedFeatureFlag]) =>
+      selectedFeatureFlag.filterMode === FILTER_MODE.INCLUDE_ALL ? 0 : tableRowCount
+    )
+  );
```

Comment:
I think this should be updated to `selectedFeatureFlag?.filterMode`, otherwise, I get a `TypeError: Cannot read properties of undefined (reading 'filterMode')` error when I navigate away from the page.